docs/unique-design-challenges.md

```diff
@@ -76,4 +77,25 @@ While not part of the core nostr protocol, zaps are popular enough that we shoul
 
 In this guide we will explore how current clients are explaining zaps (if they are) and where / how we can improve the overall user experience by doing a better job at arming the user with the necessary information.
 
-[Explore Zaps](https://nostrdesign.org/docs/how-to/zaps/)
\ No newline at end of file
+[Explore Zaps](https://nostrdesign.org/docs/how-to/zaps/)
+
+## Challenge #5: NIPs
+
+Unlike email clients (e.g. yahoo email client, microsoft outlook email client, gmail email client etc.) which (explaining in an oversimplified way) either support or don't support the email protocol, nostr apps have tens of NIPs, and potentially hundreds of NIP candidates as nostr continues to grow. 
+
+A design consequence of a multitude of opt-in NIPs means that there will be sets of apps with little overlap in supported NIPs. This means that a user of app X may not be able to interact with a user on app Y, or vice versa. This also means that unless this is surfaced to either users, they may not be aware that they cannot interact with users of another app. 
+
+## Challenge #6: Deletion across tens to hundreds of relays
+
+Event deletion in reference to [NIP-09](https://github.com/nostr-protocol/nips/blob/master/09.md]) on nostr carries a unique challenge of asking various distinct relay (i.e. database) admins to delete a particular event.
```

Comment:
Without “administrators,” in this way it appears to be a manual action, while the deletion is done by the relays (software) themselves.

Reply:
Sorry, I don't understand what you are asking with this comment. I trust you to directly modify the statement.

Reply:
I would write:

> Event deletion in reference to [NIP-09](https://github.com/nostr-protocol/nips/blob/master/09.md]) on nostr carries a unique challenge of asking various distinct relays (i.e. databases) to delete a particular event.

Make sense?

Reply:
Updated